src/js/airtable-list-builder.js

```diff
@@ -0,0 +1,286 @@
+(function ($, Drupal, drupalSettings) {
+  Drupal.behaviors.airtableListBuilder = {
+    attach: function (context, settings) {
+
+      // Get the content area
+      var $contentArea = $("#airtable-list");
+
+      // Grab the airtable variables.
+      let config = $contentArea.data('config');
+
+      var configDefault = {
+        "gutter": 10,
+        "equalHeight": false
+      }
+      config = {
+        ...configDefault,
+        ...config
+      }
+
+      for (filterKey in config.filters) {
+        config.filters[filterKey].choices = [{"key": "*", "name": "All"}];
+      }
+
+      // Load the airtable data
+      $.ajax({
+        type: "GET",
+        beforeSend: function (xhr) {
+          xhr.setRequestHeader('Authorization', 'Bearer iQ7GGRoNRcGi8iZf28xSgDgaDDUslteL');
```

Comment:
Since you have the cache proxy, can you hide this now?

Reply:
I need the api key for the proxy so no.